tests/behaviour/contracts/conditionals/conditional.sol

```diff
@@ -0,0 +1,27 @@
+pragma solidity ^0.8.10;
+
+// SPDX-License-Identifier: MIT
+
+contract WARP {
+  uint256 public x;
+
+  function simpleScalar(
```

Comment:
I'd appreciate a test case that includes conditionals where both branches are literals. Ideally one for number literals as well as some for the various kinds of string literals.